doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_reset.rst

```diff
@@ -0,0 +1,62 @@
+.. _ug_nrf54h20_architecture_reset:
+
+nRF54H20 reset behavior
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. TODO
+
+nrfutil allows for different types of reset behavior on the various cores of the nRF54H20 SoC, based on the current lifecycle state of the device.
+
+Reset kinds in LCS EMPTY
+========================
+
+.. TODO
+
+The following is the reset behavior when the LCS of the nRF54H20 SoC is in ``EMPTY`` state:
+
+.. list-table:: Reset behavior based on LCS ``EMPTY``
+   :header-rows: 1
+
+   * - Reset kind in LCS ``EMPTY``
+     - Reset behavior
+   * - ``RESET_SYSTEM``
+     - Mapped to the ``SYS_RESET_REQUEST`` bit as described in the ARM specification for the *Application Interrupt and Register Controller*.
+       This reset triggers a reset of the Secure Domain, which subsequently resets the entire system.
+       However, it cannot trigger a reset of other cores.
```

Comment:
It suggests that `RESET_SYSTEM` triggered on Secure Domain does not reset other cores.

Instead, it should state something around "This reset type is unavailable for other cores than Secure Domain"